app/modules/fileuploads/models.py

```diff
@@ -0,0 +1,174 @@
+# -*- coding: utf-8 -*-
+"""
+FileUploads database models
+--------------------
+"""
+
+from flask import current_app
+from app.extensions import db, HoustonModel
+
+import logging
+import uuid
+import os
+import shutil
+
+log = logging.getLogger(__name__)
+
+
+# TODO move these to common tus area, possibly new extension (as suggested by jp) related to "file acquisition"
```

Comment:
Would it make more sense to incorporate this into the existing tus extension?

Reply:
talk on friday was of the new FileUpload, but also (jp i think originally suggested) perhaps putting tus functionality (e.g. the stuff we have in submissions) as part of a more _general_ file acquisition extension or something.  just kinda a very rough idea.. rough draft of a draft?

Reply:
We should still talk about this after the PR is merged.. Tangentially RE: DEX-185 the ticket for investigating shared components for upload.